collection-scripts/gather_dbginfo_z

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+# dbginfo.sh is a script shipped with the s390-utils package on IBM-Z systems that collects system level information for debugging
+BASE_COLLECTION_PATH="must-gather"
+DBGINFO_PATH=${OUT:-"${BASE_COLLECTION_PATH}/node_dbginfo"}
+
+mkdir -p "${DBGINFO_PATH}"/
+
+function get_dbginfo_off_node {
+    local debugPod=""
+
+    #Get debug pod's name
+    debugPod=$(oc debug --to-namespace="default" node/"$1" -o jsonpath='{.metadata.name}')
```

Comment:
Don't push this pod to the default namespace, must-gathers already run in an isolated namespace; doing this could break (escape) our cleanup processes.

Reply:
You do this below too (multiple places to correct).

Reply:
will correct all the places.